src/plugins/intel_gpu/src/graph/crop.cpp

```diff
@@ -188,70 +188,69 @@ std::string crop_inst::to_string(crop_node const& node) {
 }
 
 crop_inst::typed_primitive_inst(network& network, crop_node const& node) : parent(network, node) {
-    const auto& ref_in_sizes = argument->reference_input;
     const auto in_layout = node.get_input_layout();
+    if (in_layout.is_dynamic())
+        return;
+    const auto& ref_in_sizes = argument->reference_input;
     const auto& offsets = argument->offsets;
     tensor null_tensor {};
     tensor value_tensor { 1, 1, 1, 1, 1 };
 
-    if (in_layout.is_static()) {
-        const auto& in_sizes = in_layout.get_tensor();
-        // Check for borders variant of crop.
-        if (ref_in_sizes.batch[0] < 0 || ref_in_sizes.feature[0] < 0 || ref_in_sizes.spatial[0] < 0 ||
-            ref_in_sizes.spatial[1] < 0 || ref_in_sizes.spatial[2] < 0) {
-            // Ignore not supported dimensions.
-            const auto rb_sizes = ref_in_sizes.negate().sub({0, 0, 0, 0, 0});
-            const auto lt_sizes = offsets.sub({0, 0, 0, 0, 0});
-
-            const auto out_sizes = in_sizes - (rb_sizes + lt_sizes);
-
-            CLDNN_ERROR_TENSOR_SIZES_LESS_THAN(node.id(),
-                                            "Left/top/lower borders",
-                                            lt_sizes,
-                                            "0 value",
-                                            null_tensor,
-                                            "Invalid border size: negative");
-            CLDNN_ERROR_TENSOR_SIZES_LESS_THAN(node.id(),
-                                            "Right/bottom/upper borders",
-                                            rb_sizes,
-                                            "0 value",
-                                            null_tensor,
-                                            "Invalid border size: negative");
-
-            CLDNN_ERROR_TENSOR_SIZES_LESS_THAN(node.id(),
-                                            "Input sizes - border sizes",
-                                            out_sizes,
-                                            "1 value",
-                                            value_tensor,
-                                            "Invalid border sizes: greater-equal input sizes");
-        }
+    const auto& in_sizes = in_layout.get_tensor();
```

Comment:
minor comment but could you please minimize the changes only required for the purpose?

Reply:
I agree. I've updated to minimize changes to only update what's needed.